src/main/java/com/google/sps/servlets/MentorSignupServlet.java

```diff
@@ -82,13 +83,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String lastName = request.getParameter("last-name");
     String username = request.getParameter("username");
     String email = userService.getCurrentUser().getEmail();
-    int major = Integer.parseInt(request.getParameter("major"));
+    int major = Utility.tryParseInt(request.getParameter("major"));
```

Comment:
optional - make the parameter strings constants too, so this would be 'request.getParameter(SqlConstants.USER_MAJOR_INDEX)' or something like that. Will make it harder to spell something wrong and create a bug :)

Reply:
Added them! Also added SQL constants for every time I query the database.